css-contain-2/Overview.bs

```diff
@@ -1994,6 +1994,37 @@ Suppressing An Element's Contents Entirely: the 'content-visibility' property {#
 		When the element becomes disconnected, the element's [=proximity to the viewport=] becomes [=not determined=].
 	</div>
 
+	<div>
+		All elements have a <dfn>currently relevant to the user</dfn> flag, initially false.
+	</div>
+
+	<div algorithm="update content relevancy for a document">
+		To <dfn export>update content relevancy for a document</dfn> given a document <var>doc</var>, and a boolean <var>isFirstIteration</var>:
+
+		1. Let <var>changed</var> be false.
+		1. For each <var>element</var> [=connected=] to doc with ''auto'' used value of ''content-visibility''
+			1. Determine proximity to the viewport for <var>element</var>.
+			1. If <var>element</var> is not [=currently relevant to the user=] and <var>element</var> is [=relevant to the user=], then:
+				2. Set <var>changed</var> to true.
+				2. Set <var>element</var>'s [=currently relevant to the user=] flag to true.
+			1. If <var>isFirstIteration</var> is false, then [=continue=].
+			1. If <var>element</var> is [=relevant to the user=], then:
+				1. If <var>element</var>'s [=currently relevant to the user=] flag is false, set <var>changed</var> to true.
```

Comment:
This condition seems to be the same as the second condition above
> If <var>element</var> is not [=currently relevant to the user=] and <var>element</var> is [=relevant to the user=], then

in fact I don't think it would hit since we update `currently relevant to the user` on line 2009. Or am I missing a subtlety here?

Reply:
Yeah I think this branch is dead effectively, but just because of the mistake you pointed out above.